src/SFML/Window/iOS/EaglContext.mm

```diff
@@ -164,8 +164,19 @@ void ensureInit()
 {
     static void* module = 0;
 
-    if (!module)
-        module = dlopen("libGLESv1_CM.dylib", RTLD_LAZY | RTLD_LOCAL);
+    const int LIB_COUNT = 3;
+    const char* LIBS[LIB_COUNT] =
```

Comment:
[SFML's Style Guide](https://www.sfml-dev.org/style.php#naming-conventions) uses `camelCase` also for constants

Reply:
Whoops, must have missed that section.